modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* Your {product-title} version is up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_.
+* You have installed the OpenShift CLI (`oc`) that matches the version for your current cluster.
+* Your `oc` client is updated to at least verion 4.13.0. 
```

Comment:
sorry for bikeshedding, but since we already ask for `Your {product-title} version is up to date to at least version 4.13.0. `   
and to have  `You have installed the OpenShift CLI ('oc') that matches the version for your current cluster.`  
mentioning `Your 'oc' client is updated to at least verion 4.13.0.` may be redundant